samples/bluetooth/mesh/dfu/target/README.rst

```diff
@@ -167,6 +167,9 @@ Only 2 options are supported by this sample:
    In this case, the device unprovisions itself before programming the new firmware.
    The unprovisioning happens before the device reboots, so if the MCUboot fails to validate the new firmware, the device will boot unprovisioned anyway.
 
+.. note::
+   To create the new Composition Data and see the :c:enum:`BT_MESH_DFU_EFFECT_UNPROV` effect, you can, for example, turn off the Friend feature in the :file:`prj.conf` file by setting the :kconfig:option:`CONFIG_BT_MESH_FRIEND` option to ``n``.
```

Comment:
@mia-ko I think you typically always tell me to do it like this?
```suggestion
   To create the new Composition Data and see the :c:enum:`BT_MESH_DFU_EFFECT_UNPROV` effect, you can, for example, turn off the Friend feature in the :file:`prj.conf` file by setting the Kconfig option :kconfig:option:`CONFIG_BT_MESH_FRIEND` to ``n``.
```

Reply:
I see both variants in our samples:
```
pavel@pavel-virtual-machine:~/nordic/ncs/git-repos/ncs/nrf/samples/bluetooth/mesh$ find -name \*.rst -exec grep -srnEH "the.*option" '{}' \;
./sensor_server/README.rst:116:The :ref:`Zephyr settings API <zephyr:settings_api>` is used to persistently store the sensor range setting for the temperature sensor, given that :kconfig:option:`CONFIG_BT_SETTING` is enabled.
./light_dimmer/README.rst:195:  The control will be re-enabled after a certain time which can be configured using the :kconfig:option:`CONFIG_BT_MESH_LIGHT_CTRL_SRV_TIME_MANUAL` option.
./dfu/target/README.rst:138:To set a new version, alter the Kconfig option :kconfig:option:`CONFIG_MCUBOOT_IMGTOOL_SIGN_VERSION` to a version that is higher than the default version of: ``"1.0.0+0"``.
./dfu/target/README.rst:148:   This sample enables the :kconfig:option:`CONFIG_BT_MESH_DFU_METADATA` option and uses the format defined by the :ref:`Bluetooth Mesh DFU subsystem<zephyr:bluetooth_mesh_dfu>`.
./dfu/distributor/README.rst:112:   To prevent an unauthenticated access to the device over SMP, it is strongly recommended to enable the :kconfig:option:`CONFIG_MCUMGR_TRANSPORT_BT_AUTHEN` option.
./dfu/distributor/README.rst:226:To prevent an unauthenticated access to the device over SMP, it is strongly recommended to enable the :kconfig:option:`CONFIG_MCUMGR_TRANSPORT_BT_AUTHEN` option.
./dfu/distributor/README.rst:231:When the model and the :kconfig:option:`CONFIG_MCUMGR_TRANSPORT_BT_AUTHEN` option are enabled while a remote device tries to read the SMP characteristics, the pairing request will be initiated and the sample will require the remote device to enter the passkey generated by the model.
./chat/chat_cli.rst:60:   The length of the text string can be configured at the compile-time using :ref:`CONFIG_BT_MESH_CHAT_CLI_MESSAGE_LENGTH <CONFIG_BT_MESH_CHAT_CLI_MESSAGE_LENGTH>` option.
./chat/chat_cli.rst:66:   The length of the text string can be configured at the compile-time using :ref:`CONFIG_BT_MESH_CHAT_CLI_MESSAGE_LENGTH <CONFIG_BT_MESH_CHAT_CLI_MESSAGE_LENGTH>` option.
./light_switch/README.rst:216:The configuration overlay :file:`overlay-lpn.conf` enables the LPN feature, and alters certain configuration options to further lower the power consumption.
```